CMakeLists.txt

```diff
@@ -76,6 +76,16 @@ if(BUILD_TESTING)
     sensor_msgs
     tf2_ros
   )
+  if(MSVC)
+    # CMake on Windows places executables in ${CMAKE_CURRENT_BINARY_DIR}/${RELEASE_TYPE},
```

Comment:
I haven't noticed similar workarounds in other packages. What is different here?

Reply:
Instead, I think we can use a generator expression like we do in [test_communication](https://github.com/ros2/system_tests/blob/7f018b29ebf3175bdbadcedfe4e45e25b8e08427/test_communication/CMakeLists.txt#L85). E.g.

```diff
- ${CMAKE_CURRENT_BINARY_DIR}/test_two_links_moving_joint
+ $<TARGET_FILE:test_two_links_moving_joint>
```

Reply:
Ah, that was the bit I didn't know.  OK, switched to TARGET_FILE, will run CI one more time.